ui/scripts/cloudStack.js

```diff
@@ -329,7 +325,6 @@
                         g_regionsecondaryenabled = null;
                         g_loginCmdText = null;
                         
-                        $.cookie('JSESSIONID', null);
                         $.cookie('sessionkey', null);
```

Comment:
Indeed Erik, thanks for pointing that out. I've updated the PR.

On Tue, May 26, 2015 at 11:46 PM, Erik Weber <notifications@github.com>
wrote:

> In ui/scripts/cloudStack.js
> <https://github.com/apache/cloudstack/pull/308#discussion_r31083230>:
>
> > @@ -329,7 +325,6 @@
> >                          g_regionsecondaryenabled = null;
> >                          g_loginCmdText = null;
> >
> > -                        $.cookie('JSESSIONID', null);
> >                          $.cookie('sessionkey', null);
>
> I'm no UI/Javascript expert, but should these include hostname as well? If
> so, same goes for L351+
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cloudstack/pull/308/files#r31083230>.
>